### ISSUE TITLE
Incorrect dependency

### ISSUE THREAD

**Issue by RickEyre** [scotiabank/eslint-config-scotiabank#8]
When installing this warning pops up

```bash
warning "eslint-plugin-import@2.2.0" has incorrect peer dependency "eslint@2.x - 3.x".
```

We should upgrade `eslint-plugin-import` to be compatible with the `eslint` version.

**Comment by RickEyre** [comment]
@tshen8, was this fixed by #12 ?

**Comment by tshen8** [comment]
The latest `eslint-plugin-import` has `"eslint": "2.x - 4.x"` listed in its peer dependencies.
This issue can be closed.

**Comment by RickEyre** [comment]
Fixed by #12
